engine/apps/mobile_app/tasks.py

```diff
@@ -274,11 +274,14 @@ def _get_youre_going_oncall_fcm_message(
         schedule, schedule_event, mobile_app_user_settings
     )
 
+    # safe to cast this as the model has a default value assigned to it
+    info_notification_sound_name = typing.cast(str, mobile_app_user_settings.info_notification_sound_name)
```

Comment:
need to cast this because `mobile_app_user_settings.info_notification_sound_name` has a type of `str | None` and you can't add `None` + a `str` (down on line 284 and 297)

Reply:
maybe `f"{info_notification_sound_name}{MobileAppUserSettings.ANDROID_SOUND_NAME_EXTENSION}"` would make sense here?